### ISSUE TITLE
Filter homes via StructureID

### ISSUE THREAD

**Issue by justinshuls** [Brandawg93/homebridge-nest-cam#47]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by willjasen** [comment]
I second this, I have 3 structures and only want to see my own home's cameras.

**Comment by Brandawg93** [comment]
I've created a test version of the plugin with this enhancement. You can install it via `npm -g i homebridge-nest-cam@test`. In your `config.json` file, under options, add an array named `"structures"`. It should look something like this:
```
{
    "platform": "Nest-cam",
    "ffmpegCodec": "libx264",
    "options": {
      "motionDetection": true,
      "streamingSwitch": true,
      "disableAudio": false,
      "structures": ["TheNameOfYourStructure"]
    },
    "googleAuth": {
        "issueToken": "https://accounts.google.com/o/oauth2/iframerpc?action=issueToken...",
        "cookies": "SMSV=ADHTe...",
        "apiKey": "AIzaS..."
    }
}
```

The array should include the exact name of all structures' cameras that you would like to see in the Home app. You can find the name of the structures associated with your Nest account in the settings of the Nest app.

Please let me know if this works for you and I will add it to the latest version of the plugin!

**Comment by valiquette** [comment]
I was able to test this and it works as expected, I did assume Structure ID at first, it is not clear in the readme that you are looking for the Structure Name. I did not catch that until reading your comment in this thread. The readme could be more specific or the config could be, The config just said Structure. It did take a few restarts to clear out the un wanted cameras. I probably should have cleared the cache first, that might have made a difference. Thanks this is a much need enhancement.
